examples/lambda-example/src/main/resources/application.properties

```diff
@@ -12,7 +12,7 @@ web.component.scan=com.accenture
 #
 # If you don't need REST endpoints, you can disable it with rest.automation=false
 #
-rest.server.port=8085
+rest.server.port=8081
```

Comment:
Please keep the example app config intact

Reply:
Requested change has been made